Makefile

```diff
@@ -1,4 +1,4 @@
-GO = go # if using docker, should not need to be installed/linked
+GO = /usr/local/go/bin/go
```

Comment:
i absolutely not sure it's cross-platform

Reply:
Sorry this my mistake I have commit a  local file.

Reply:
Removed the mistake